templates/repo/settings/protected_branch.tmpl

```diff
@@ -149,6 +149,13 @@
 						<p class="help">{{ctx.Locale.Tr "repo.settings.dismiss_stale_approvals_desc"}}</p>
 					</div>
 				</div>
+				<div class="field">
+					<div class="ui checkbox">
+						<input name="ignore_stale_approvals" type="checkbox" {{if .Rule.IgnoreStaleApprovals}}checked{{end}}>
```

Comment:
Could we check both `DismissStaleApprovals` and `IgnoreStaleApprovals`?

Reply:
Do you suggest to put `{{if or .Rule.IgnoreStaleApprovals .Rule.DismissStaleApprovals}}`?

Or do you question whether it makes sense to enable both options?

Dismissed reviews are always ignored when counting approvals:
https://github.com/go-gitea/gitea/blob/69f2b698c5000a566bc04267f7e9bd387b880109/models/issues/pull.go#L833